### PR TITLE
63 converting and revising readmemd to readmerst

### PR DESCRIPTION
**Pull request by jonglee1**

### CODE REVIEW

**Review by longshuicy — APPROVED** [review]

**Comment by ywkim312** [comment]
@jonglee1 Setup.py also has to be updated to read the readme.rst.

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+============
+
+**pyincore-viz** is a Python project that provides visualization and other utilities for use with **pyincore**
+The development is a part of NIST sponsored IN-CORE (Interdependent Networked Community Resilience Modeling
+Environment) initiative. 
+
+Mac specific notes
+------------------
+
+- We use `matplotlib` library to create graphs. There is a Mac specific installation issue addressed at `here1 <https://stackoverflow.com/questions/4130355/python-matplotlib-framework-under-macosx>`_ and
```
Comment (navarroc) [inline README.rst:11]: should here1 and here2 just be "here" and "here" or was it intentional to name them separately as here1 and here2?

Comment (jonglee1) [reply README.rst:11]: rst doesn't like here and here. It says that it has duplicated reference

Comment (ywkim312) [reply README.rst]: @jonglee1 can you update version in setup.py as well?

**Review by navarroc — APPROVED** [review]

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+-----------------------
+
+Installing **pyincore-viz** with pip is **NOT supported** by IN-CORE development team.
+Please use pip for installing pyincore-viz at your discretion. 
+
+**Installing pyincore-viz with pip is only tested on linux environment.**
+
+**Prerequisite**
+
+* GDAL C library must be installed to install pyincore. (for Ubuntu, **gdal-bin** and **libgdal-dev**)
```
[the remark this answers is not in the document]
Comment (ywkim312) [reply README.rst]: I will fix it in release

**Pull request merged**